src/Common/src/Interop/Unix/libc/Interop.Permissions.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+internal static partial class Interop
+{
+    internal static partial class libc
+    {
+        [Flags]
+        internal enum Permissions
```

Comment:
I think they should be in the library type if they're defined by header files from that library, which they are in this case.  I'll make sure the guidance is clear on that (unless you disagree).